<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScreen</class>
 <widget class="QWidget" name="LoginScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#seConnecter{
background-color: #FF8A5B;
color:#FCEADE;
border-style: solid;
border-radius:13px;
}
QPushButton#seConnecter:pressed{
background-color: #25CED1;

}
QPushButton#oublie{
background-color: white;
color:#25CED1;
border-style: solid;

}
QLabel{
color:#2E382E;
}
QLineEdit{
border:1px solid #25CED1;
}
QLineEdit:focus {
border: 1px solid #FF8A5B;
}
QWidget{
background-color:white;
}</string>
  </property>
  <widget class="QLineEdit" name="lineID">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>156</y>
     <width>252</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Identifiant</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMDP">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>213</y>
     <width>252</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Mot de passe</string>
   </property>
  </widget>
  <widget class="QLabel" name="labID">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>139</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Identifiant :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labMDP">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>196</y>
     <width>74</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mot de passe :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="seConnecter">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>102</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
    </font>
   </property>
   <property name="text">
    <string>Se Connecter</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labConn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>112</y>
     <width>300</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>CONNECTION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>31</y>
     <width>193</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Images.qrc">:/login/images/logoComp.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="oublie">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Mot de Passe Oubli√©?</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
